emitter/src/tests/action.spec.ts

```diff
@@ -37,12 +37,29 @@ describe('Actions', () => {
       setState(getState() * 2);
     }
 
+    @Receiver()
+    public static dontChange() {
+      CounterStore.increment.emit();
+      CounterStore.decrement.emit();
+    }
+
     @Receiver()
     public static throwError() {
       return throwError(new Error('Whoops!'));
     }
   }
 
+  class CounterStore {
```

Comment:
Where decorator State?

Reply:
It is not a state so no need to use decorator. It is just a helper class to emit actions without using component.